includes/roslyn-installation.md

```diff
@@ -0,0 +1,29 @@
+## Installation instructions 
+
+There are two different ways to find the **.NET Compiler Platform SDK** in the **Visual Studio Installer**:
+
+### Install using the Workloads view
+
+The .NET Compiler Platform SDK is not automatically selected as part of the Visual Studio extension development workload. You must select it as an optional component.
+
+1. Run **Visual Studio Installer** 
+1. Select **Modify** 
+1. Check the **Visual Studio extension development** workload.
+1. Open the **Visual Studio extension development** node in the summary tree.
+1. Check the box for **.NET Compiler Platform SDK**. You'll find it last under the optional components.
+
+Optionally, you'll also want the **DGML editor** to display graphs in the visualizer:
+
+1. Open the **Individual components** node in the summary tree.
+1. Check the box for **DGML editor**
+
+### Install using the Individual components tab
+
+1. Run **Visual Studio Installer** 
+1. Select **Modify** 
+1. Select the **Individual components** tab 
+1. Check the box for **.NET Compiler Platform SDK**. You'll find it last under the **Compilers, build tools ,and runtimes** section.
+
+Optionally, you'll also want the **DGML editor** to display graphs in the visualizer:
```

Comment:
Sorry for being late but it seems that this is the wrong option 

https://github.com/dotnet/roslyn-sdk/issues/65

The issue mentions *Code map* is the right choice.

Reply:
@MaStr11 Thanks for pointing this out. I did experiments, and only the DGML editor is needed. The Code Map feature depends on the DGML editor, so that will work too.